operator/flags.go

```diff
@@ -32,12 +32,15 @@ func init() {
 
 	flags := rootCmd.Flags()
 
-	// AWS dedicated flags
-	flags.Int(option.AWSClientBurst, defaults.AWSClientBurst, "Burst value allowed for the AWS client used by the AWS ENI IPAM")
-	option.BindEnv(option.AWSClientBurst)
-
-	flags.Float64(option.AWSClientQPSLimit, defaults.AWSClientQPSLimit, "Queries per second limit for the AWS client used by the AWS ENI IPAM")
-	option.BindEnv(option.AWSClientQPSLimit)
+	flags.Int(option.AWSClientBurstDeprecated, defaults.IPAMAPIBurst, "")
```

Comment:
Was the description of the flag removed by accident or on purpose?

Reply:
Yes, because it is unused. I didn't want to keep the outdated description and also didn't want to duplicate it.